docs/source/quickstart.mdx

```diff
@@ -83,23 +184,105 @@ outputs = pipeline(
 )
 ```
 
+## Training on Gaudi
```

Comment:
Maybe move training before the diffusion section?

Reply:
Done